### PR TITLE
48598_CSS_fix

### PR DESCRIPTION
**Pull request by asifbuetcse**
course
couse_show
report_create
sup_home
users_index

### CODE REVIEW

**Comment by yasin-buet** [comment]
Ok

File: resources/views/trainee/users/index.blade.php [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
+                    Start Date: {!! $course->start_date !!} <br>
+                    End Date: {!! $course->end_date !!} 
+                </p>
+            @endforeach
+        </div>
     </div>
+</div>
 @endsection
+
+
```
Comment (dinhhoanglong91) [inline resources/views/trainee/users/index.blade.php]: remove unnecessary lines

File: resources/views/trainee/courses/index.blade.php [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+        </table>
+    </div>
 </div>
 <center>
     {!! $courses->render() !!}
 </center>
-@endsection
\ No newline at end of file
+@endsection
+
```
Comment (dinhhoanglong91) [inline resources/views/trainee/courses/index.blade.php]: remove it

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**